pkg/cmd/cli/cmd/request_project.go

```diff
@@ -46,8 +50,8 @@ func NewCmdRequestProject(name, fullName, oscLoginName, oscProjectName string, f
 		Short: "request a new project",
 		Long:  fmt.Sprintf(requestProjectLongDesc, fullName, oscLoginName, oscProjectName),
 		Run: func(cmd *cobra.Command, args []string) {
-			if !options.complete(cmd) {
-				return
+			if err := options.complete(cmd, f, out); err != nil {
+				glog.Fatal(err)
```

Comment:
Yes, also, please do a sweep of all other uses of glog.Fatal in the user facing tools.